src/blocks/product_card.css

```diff
@@ -0,0 +1,88 @@
+.product-card {
+  width: 200px;
+  background-color: white;
+  border: 1px solid #f3f3f3;
+  border-radius: 6px;
+  padding: 32px 16px 16px;
+}
+
+.product-card__img {
+  display: block;
+  background-image: url(../images/imac.jpeg);
+  background-repeat: no-repeat;
+  background-size: 160px 134px;
+  height: 134px;
+  width: 160px;
+  margin: 0 auto;
+}
+
+.product-card__name {
+  font-weight: 500;
+  color: #060b35;
+  line-height: 18px;
+  margin-top: 40px;
+  font-size: 12px;
```

Comment:
Why did you put margin between text rules?

Reply:
he was so lonely...